### ISSUE TITLE
Is WatsonWebserver built based on HTTP.SYS technology?

### ISSUE THREAD

**Issue by sqlnew** [jchristn/WatsonWebserver#109]
I would like to know what technology is used to implement the underlying Watson Web server. It should support high concurrency. Is it based on Microsoft's HTTP. SYS technology?

**Comment by jchristn** [comment]
Yes, it is a wrapper around the capabilities provided by http.sys.
